### ISSUE TITLE
New Insight: Gender Analysis (EOY)

### ISSUE THREAD

**Issue by adampash** [ThinkUpLLC/ThinkUp#2153]
# One-liner

Gender breakdown of commenters and likers on your Facebook status updates this year.

![Insight preview](http://files.adampash.com/s/User_Name_s_status_updates_resonated_with_women_in_2014_1A0AA665.png)
[Insight preview](https://thinkup.thinkup.com/insight_tester.php?headline=User%20Name%27s%20status%20updates%20resonated%20with%20women%20in%202014&body=This%20year%2C%2012%20likes%20and%20comments%20on%20User%20Name%27s%20status%20updates%20were%20by%20people%20who%20identify%20as%20female%2C%20compared%20to%208%20by%20people%20who%20identify%20as%20male.%0A&button=Action!&avatar=https%3A%2F%2Fpbs.twimg.com%2Fprofile_images%2F1096005346%2F1_normal.jpg&hero=https%3A%2F%2Fwww.thinkup.com%2Fassets%2Fimages%2Finsights%2F2014-05%2Fsubway.jpg&show-button=false&show-avatar=false&show-emphasis=true&show-hero=false&show-wide=true&color=pea&embed=show-pie&preview=1)
# Audience for the insight

Facebook only.
# How often this insight runs
- [ ] Always (triggered by a data event, check the data and update insight on every crawl)
- [ ] Daily (check the data once a day, don't regenerate every crawl)
- [ ] Weekly on a fixed day (specify day of week per network to space these out)
- [ ] Monthly on a fixed day with bonus magic day within first 2 weeks of use (specify day of month per network to space these out)
- [ ] Annual on a fixed date (specify day of year)
- [x] Annual every 365 days from first appearance

December X (will need to be same day as most retweeted, which is currently Dec 1, but should probably change).
# Headline
- %username's status updates resonated with %gender in %year
# Body
- This year, 12 likes and comments on User Name's status updates were by people who identify as female, compared to 8 by people who identify as male.

Basically, body and headline copy should follow same logic as body and headline copy for the monthly version of this insight. We don't need more than one variation of the headline (but should account for "resonated with both genders" for equal case), and likewise with body copy.
# Criteria and logic

Logic is the same as monthly version of this insight.
## Emphasis
- [x] High
- [ ] Medium
- [ ] Low
# Included elements
- [x] Headline
- [x] Text
- [ ] Header image (image off to the left in side-by-side style insights)
- [ ] Hero image (giant image on top)
- [ ] List of user(s)
- [ ] List of post(s)
- [ ] List of link(s)
- [ ] Action button (please specify button label and URL button should link to)
- [ ] Line chart
- [ ] Bar chart
- [x] Other viz
- [ ] Other graphic treatment

Pie chart.
